libsweep/src/dummy.cc

```diff
@@ -34,6 +34,65 @@ void sweep_error_destruct(sweep_error_s error) {
   delete error;
 }
 
+// ------------------ Blocking Methods (blocking) ---------------- //
```

Comment:
reorganize the methods to match sweep.h/sweep.cc and remove the outdated comments